nessie/jobs/generate_merged_student_feeds.py

```diff
@@ -239,7 +239,7 @@ def generate_student_profile_feed(self, feed_elements, rows, advisors):
 
             for plan in sis_profile.get('plans', []):
                 if plan.get('status') == 'Active':
-                    rows['student_majors'].append(encoded_tsv_row([sid, plan['program'], plan['description']]))
+                    rows['student_majors'].append(encoded_tsv_row([sid, plan.get('program', None), plan['description']]))
```

Comment:
@lyttam – Looks good. To verify, the `'description'` key will always be present? No need for `.get(...)`?

Reply:
@johncrossman It should always be preset - at least that's how it's been working.  I could add a `.get` for extra security.

Reply:
Amended.